yabt/target_extraction_test.py

```diff
@@ -22,6 +22,8 @@
 """
 
 
+from os.path import join
```

Comment:
I wouldn't import join like that - it makes the code less readable since it is not clear which type of join is it.
```
from os import path
...
path.join(...
```
would be clearer.

Reply:
fixed